src/test/java/Julia/EatStreet/Tests/TestAdressInput.java

```diff
@@ -0,0 +1,24 @@
+package Julia.EatStreet.Tests;
+
+import ClasesToAllUs.TestInit;
+import Julia.EatStreet.Pages.HomePage;
+import org.testng.annotations.Test;
+
+public class TestAdressInput extends TestInit {
+
+    private void novigateEatStreet() {
+        driver.get("https://eatstreet.com/");
+    }
+
+    @Test
+    public void testEatStreet1() {
+        HomePage homePage = new HomePage(driver);
+        novigateEatStreet();
+        homePage.getGoItBtn().click();
+        homePage.getAdressInput().sendKeys("Miami");
+        homePage.getFedBtn().click();
+        homePage.getFedBtn().click();
+        sleep(5);
+
```

Comment:
Why sleep() in the end test?
Were Assert?

Reply:
ok